.gitignore

```diff
@@ -1,20 +1,17 @@
-res/courses/
-res/fastimetable2014.csv
-timetable.html
-timetable2014.csv
-__pycache__/*
-*.xlsx
-res/courses/*
-enrol.txt
-res/courses-copy/
-*.pyc
-courses.tar.gz
-out/
-*.exe
-*.hi
-*.o
+# cabal sandbox
+.cabal-sandbox/
+cabal.sandbox.config
+
+# local database
 *.sqlite3
-style/
-SVGGen.hs
-res/graphs/CSC/csc_graph.svg
-hs/dist/
\ No newline at end of file
+*.sqlite3-shm
+*.sqlite3-wal
+
+# generated build files
+/hs/dist/
+
+# generated CSS
+/public/style/
+
+# generated resources
+/public/res/graphs/
```

Comment:
We need to keep this for now. The graphs made manually (in yEd and Inkscape) are currently also stored in this directory. We can move them to a separate location.